release/nightly_tests/dataset/shuffle_ingest.py

```diff
@@ -0,0 +1,79 @@
+import time
+import ray
+import argparse
+
+DUMMY_ROW = {
+    "key": 0,
+    "embeddings_name0": 1527,
+    "embeddings_name1": 175,
+    "embeddings_name2": 8,
+    "embeddings_name3": 5,
+    "embeddings_name4": 5,
+    "embeddings_name5": 687,
+    "embeddings_name6": 165,
+    "embeddings_name7": 10,
+    "embeddings_name8": 137,
+    "embeddings_name9": 597,
+    "embeddings_name10": 1574,
+    "embeddings_name11": 78522,
+    "embeddings_name12": 283941,
+    "embeddings_name13": 3171,
+    "embeddings_name14": 39560,
+    "embeddings_name15": 718571,
+    "embeddings_name16": 73699,
+    "one_hot0": 1,
+    "one_hot1": 30,
+    "labels": 0.3801173847541949,
+    "__index_level_0__": 0
+}
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(description="Eric Example")
```

Comment:
rename this description to maybe "Test script to evaluate distributed shuffling with consumers".